prometheus-net.tests/MetricsTests.cs

```diff
@@ -124,12 +127,10 @@ public void gauge_collection()
             metrics[0].gauge.value.ShouldEqual(3.8);
         }
 
-      
-
         [Test]
         public void histogram_tests()
         {
-            Histogram histogram = Metrics.CreateHistogram("hist1", "help", new []{ 1.0, 2.0, 3.0});
+            Histogram histogram = Metrics.CreateHistogram("hist1", "help", new []{ 1.0, 2.0, 3.0, double.PositiveInfinity});
```

Comment:
Added infinity here to test that infinity doesn't get added twice as there was no test for that. Other (new) test cases test without the infinity item to make sure it does get added as expected